core/src/main/java/com/graphhopper/routing/weighting/custom/CustomProfile.java

```diff
@@ -18,6 +35,7 @@ public CustomProfile(String name) {
     }
 
     public CustomProfile setCustomModel(CustomModel customModel) {
+        customModel.__internal();
```

Comment:
Not sure if I like this too much :smile: but the static size map was also not optimal.

Reply:
Maybe leave a comment so its clear what this is for?